typeDefs.js

```diff
@@ -126,11 +126,17 @@ const typeDefs = gql`
     available: Boolean
     additionalPrice: Int
   }
+
   type Error {
     statusCode: Int
     message: String
   }
 
+  type Products {
```

Comment:
Call it `ProductsPaginatedResult` to show by name that it is a result with pagination

Reply:
renamed